testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/ClientPolicyBasicsTest.java

```diff
@@ -105,15 +80,22 @@
 import org.keycloak.testsuite.services.clientpolicy.condition.TestRaiseExeptionConditionFactory;
 import org.keycloak.testsuite.util.MutualTLSUtils;
 import org.keycloak.testsuite.util.OAuthClient;
+import org.keycloak.util.JsonSerialization;
 
+import javax.ws.rs.BadRequestException;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
 import java.security.*;
 import java.security.spec.InvalidKeySpecException;
 import java.security.spec.PKCS8EncodedKeySpec;
 import java.security.spec.X509EncodedKeySpec;
 import java.util.*;
+import java.util.function.Consumer;
 
+import static org.junit.Assert.*;
 import static org.keycloak.testsuite.AbstractTestRealmKeycloakTest.TEST_REALM_NAME;
-import org.keycloak.util.JsonSerialization;
+import static org.keycloak.testsuite.admin.AbstractAdminTest.loadJson;
+import static org.keycloak.testsuite.admin.ApiUtil.findUserByUsername;
 
 @EnableFeature(value = Profile.Feature.CLIENT_POLICIES, skipRestart = true)
 public class ClientPolicyBasicsTest extends AbstractKeycloakTest {
```

Comment:
IMO, it might be better to test the migrated 7 executors by test methods implemented on `ClientRegistrationPoliciesTest` which tests the original client registration policies, which means that these tests would be also migrated onto `ClientPolicyBasicsTest `. To do so, we can prove that the migrated 7 executors do the same thing done by the original 7 client registration policies.

Reply:
I've already covered migrated executors by integration tests in `ClientPolicyBasicsTest `  base on tests from `ClientRegistrationPoliciesTest`.